### PR TITLE
docs: Revise before_you_begin.rst

### PR DESCRIPTION
**Pull request by pierwill**
## Status

Ready for review

## Description of Changes

Revises the Before You Begin guide.

(Note to reviewers: I've re-lineated the paragraphs to 80 characters wide, nuking the diff. Since this doc is short, I hope the inconvenience is minor.)

## Testing

- `make docs`
-  open http://localhost:8000/before_you_begin.html

##

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: docs/before_you_begin.rst [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
+
+SecureDrop is a technical tool. It is designed to protect journalists and
+sources, but no tool can guarantee safety. This guide will instruct you in
+installing and configuring SecureDrop, but it does not explain how to use it
+safely and effectively. Put another way: at the end of this guide, you will have
+built a car; you will not know how to drive. The :ref:`Deployment Guide
+<deployment>` contains best practices for working with SecureDrop. Make sure to
+read it after completing the installation.
+
+Installing SecureDrop is an extended manual process which requires extensive
```
Comment (dachary) [inline docs/before_you_begin.rst]: Non-native english speaker remark, feel free to disregard: **extensive** following **extended** sounds a little strange.

File: docs/before_you_begin.rst [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
+version numbers, usernames, hostnames, or IP addresses, make sure to use the
+appropriate values for your instance.
+
+Once you're familiar with SecureDrop, have made your plan, ensured your
+organization is ready to follow through, and assembled the necessary hardware,
+you're ready to begin.
+
+----
+
+.. warning:: Remember to destroy the SecureDrop Installation Worksheet after the
```
Comment (dachary) [inline docs/before_you_begin.rst]: Nit: a link to **SecureDrop Installation Worksheet** would be nice here.

File: docs/before_you_begin.rst [hunk 3]
```diff
@@ -1,39 +1,45 @@
...
+Worksheet`_.  It is **critical** that you destroy this worksheet when your
+installation is complete and all of your passphrases have been safely stored in
+a password manager.
+
+When running commands or editing configuration files that include filenames,
+version numbers, usernames, hostnames, or IP addresses, make sure to use the
+appropriate values for your instance.
+
+Once you're familiar with SecureDrop, have made your plan, ensured your
+organization is ready to follow through, and assembled the necessary hardware,
```
Comment (dachary) [inline docs/before_you_begin.rst]: Nit: we have **required hardware** above, is there a reason to change to **necessary** here?

**Comment by pierwill** [comment]
@dachary : Thanks for your notes. I'll add a link in the worksheet warning. I'd like to leave the other two as-is for now, if you don't mind.

**Pull request merged**